### PR TITLE
Add initial implementation

### LINKED ISSUE #2: Add initial implementation

### PR DESCRIPTION
**Pull request by lhorie**
Resolves #2

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  extends: [require.resolve('eslint-config-fusion')],
+  rules: {
+    'no-unused-vars': [
+      'error',
+      {vars: 'all', args: 'none', varsIgnorePattern: '^h$'},
```
Comment (rtsao) [inline .eslintrc.js]: Is this mean to be here? I assume this has to do with JSX?

Comment (lhorie) [reply .eslintrc.js]: I think it might be an old config. It seems to be a bad copy-paste from some preact thing

**Comment by lhorie** [comment]
!merge

File: README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+const Hello = () => <div>Hello</div>
+
+export default function() {
+  return new App(<Hello />);
+}
+```
+
+---
+
+### API
```
Comment (rtsao) [inline README.md]: Docs for HOC?

Comment (ganemone) [reply README.md]: Honestly I think we should maybe not document it at all. I'm not convinced we want to expose this interface

Comment (lhorie) [reply README.md]: my 2 cents: document, and if we change it, do it formally via a changelog/version bump

**Pull request merged**